todo_app/app.py

```diff
@@ -11,9 +11,19 @@
 from functools import wraps
 import requests
 
+from loggly.handlers import HTTPSHandler
+from logging import Formatter
+
 def create_app():
     app = Flask(__name__)
     app.config.from_object('todo_app.flask_config.Config')
+    app.logger.setLevel(os.getenv('LOG_LEVEL'))
```

Comment:
Minor, but I think this is an environment variable where setting a default for when it's missing would probably be sensible, rather than necessarily having the app crashing
```suggestion
    app.logger.setLevel(os.getenv('LOG_LEVEL', 'INFO'))
```